checks.d/postgres.py

```diff
@@ -177,7 +177,7 @@ class PostgreSql(AgentCheck):
             ('schemaname', 'schema')
         ],
         'metrics': {
-            'pg_stat_user_tables': ('postgresql.total_tables', GAUGE),
+            'pg_stat_user_tables': ('postgresql.table.count', GAUGE),
```

Comment:
Agreed. I checked and no such screens and monitors exist right now. We can recheck before we release.